packages/components/addon/components/hds/page-header/title.hbs

```diff
@@ -2,4 +2,4 @@
   Copyright (c) HashiCorp, Inc.
   SPDX-License-Identifier: MPL-2.0
 }}
-<h1 class="hds-foreground-strong hds-typography-display-500 hds-page-header__title" ...attributes>{{yield}}</h1>
\ No newline at end of file
+<h1 class="hds-foreground-strong hds-typography-display-500 hds-page-header__title-text" ...attributes>{{yield}}</h1>
```

Comment:
I assume we're not worried about this class name change because we don't have users of this component _yet_